locales/sv/zodiac.json

```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Väduren",
+		"element": 0,
+		"stone": "blodsten"
```

Comment:
In Swedish, is it "blodsten" or "heliotrop"?

Reply:
heliotrop and blodsten refers to the same kind of stone, so either or is fine.